processor/blockprocessor/initialize.go

```diff
@@ -26,21 +24,9 @@ import (
 )
 
 // InitializeLedgerSimple executes the migration core functionality.
-func InitializeLedgerSimple(logger *log.Logger, round uint64, opts *idb.IndexerDbOptions) error {
-	ctx, cf := context.WithCancel(context.Background())
+func InitializeLedgerSimple(ctx context.Context, logger *log.Logger, round uint64, opts *idb.IndexerDbOptions) error {
```

Comment:
Is this function supposed to block until initialization is complete? It looks like it starts fetcher and then runs in the background.